torch/csrc/jit/exception_message.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <c10/util/Exception.h>
+#include <stdexcept>
+
+namespace torch {
+namespace jit {
+
+struct ExceptionMessage {
```

Comment:
Can you add a comment explaining what purpose this serves? Why cut out the c10 backtarce is not obvious I think.